health_check/views.py

```diff
@@ -126,8 +126,7 @@ def _run(plugin):
                 return self.render_to_response(context, status=status_code)
             elif media.mime_type in ('application/json', 'application/*'):
                 return self.render_to_response_json(plugins, status_code)
-            else:
-                return self.render_to_response(context, status=status_code)
+        return HttpResponse('Only HTML and JSON responses are supported', status=406, content_type='text/plain')
```

Comment:
```suggestion
        return HttpResponse('Not Acceptable: Supported content types: plain/html, application/json', status=http.HTTPStatus.NOT_ACCEPTABLE, content_type='text/plain')
```

Reply:
Thanks for the suggestion. I applied it with a minor tweak.